images/build/Dockerfile

```diff
@@ -298,6 +298,10 @@ COPY --from=mcr.microsoft.com/oryx/php-build-7.3:20190325.6 /opt /opt
 COPY --from=buildscriptbinaries /opt/buildscriptgen/ /opt/buildscriptgen/
 RUN ln -s /opt/buildscriptgen/GenerateBuildScript /usr/local/bin/oryx
 
+# Copy applicationinsightloader for node
+COPY images/build/node/oryxappinsightloader.js /tmp
+RUN chmod +x /tmp/oryxappinsightloader.js
+
```

Comment:
I will remove this to create the file dynamically when all conditions are satisfied